sonic-net/sonic-buildimage
# [FRR] Fixing the advertisement of static ipv6 route

#### Why I did it

To fix an issue - https://github.com/FRRouting/frr/issues/13587

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it

Ported a fix from FRR - https://github.com/FRRouting/frr/pull/13875

#### How to verify it

Deploy image on T0, verify that Loopback IPv6 network is advertised to arista:

```
ARISTA01T1>show ipv6 route fc00:1::

VRF: default
Routing entry for fc00:1::
Codes: C - connected, S - static, K - kernel, O3 - OSPFv3, B - BGP, R - RIP, A B - BGP Aggregate, I L1 - IS-IS level 1, I L2 - IS-IS level 2, DH - DHCP, NG - Nexthop Group Static Route, M
- Martian, DP - Dynamic Policy Route, L - VRF Leaked, RC - Route Cache Route

 B        fc00:1::/64 [200/0]
           via fc00::1, Port-Channel1

ARISTA01T1>ping fc00:1::32
PING fc00:1::32(fc00:1::32) 72 data bytes
80 bytes from fc00:1::32: icmp_seq=1 ttl=64 time=1.54 ms
80 bytes from fc00:1::32: icmp_seq=2 ttl=64 time=1.11 ms
80 bytes from fc00:1::32: icmp_seq=3 ttl=64 time=1.17 ms
80 bytes from fc00:1::32: icmp_seq=4 ttl=64 time=1.17 ms
80 bytes from fc00:1::32: icmp_seq=5 ttl=64 time=1.18 ms

--- fc00:1::32 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 5ms
rtt min/avg/max/mdev = 1.117/1.241/1.547/0.155 ms, ipg/ewma 1.428/1.390 ms
```



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
